src/sass/vic-v2.scss

```diff
@@ -401,3 +405,22 @@ input[type=range] {
   clear: both;
   margin: 0 0 1.5em 0;
 }
+/* USWDS overrides */
+.usa-accordion, .usa-accordion-bordered {
```

Comment:
The accordion styles are such a pain to work with.

Reply:
Agreed.